### DISCUSSION TITLE
Proceso de primera práctica.

### DISCUSSION THREAD

**Discussion by maxkario** [Launch-X-Latam/MisionFrontEnd#191] [Show and tell]
Saludos a todos. 
Quiero mostrar el proceso que llevo de la practica AbogaBot. 
Quizás no lleve mucho pero es un proceso de tiempo y análisis. Quiero destacar que estoy utilizando **_Notion_** e **_Illustrator_** para esta práctica, ya que son software que se utilizar y me facilitan el trabajo.
Aprecio sus consejos y observaciones. 
Gracias de antemano. 

[Enlace del proceso práctica 1](https://github.com/maxkario/Practica-1-Caso-AbogaBot)

**Comment by WerlynRodriguez** [comment]
Me gusta mucho como te quedó la documentación en Notion. Creo que me gustaría hacerlo ahí también jsjsjs. Sobre el diseño UI me imagino que debería ser un poco más claro, sobre qué cosa es, cada elemento... por lo que ahora solo lo veo como una idea abstracta.

**Reply by maxkario** [reply]
El diseño UI ya es la parte de interactividad de usuario. La parte final con el diseño implementado. Pero en mi caso tengo que diseñar cada botón y buscar los iconos y demás elementos. Y lleva su tiempo.

**Comment by kerizrh** [comment]
Igual me gusta cómo se mira la documentación en ese sitio, creo que de igual manera meteré las mías ahí.

**Comment by LauraHenao27** [comment]
Me encanta la forma en que lo organizaste, y le sacaste el provecho a Notion.

**Reply by maxkario** [reply]
Realmente es una herramienta excelente para mucha cosas. Aprendi a usarla este año y aun no la domino como se debe.
